metering/metering-installing-metering.adoc

```diff
@@ -5,10 +5,25 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+Before installing metering into your cluster, review the following sections.
+
+[WARNING]
+====
+Installing Metering on an Openshift FIPS-enabled cluster with TLS enabled will fail while attempting to generate TLS certificates for the user.
+
+See link:https://bugzilla.redhat.com/show_bug.cgi?id=1787745[BZ#1787745] for details.
+====
+
 On {product-title} 4 clusters, metering can be installed through OperatorHub. If you are using an earlier version of {product-title} or a Kubernetes cluster without OpenShift, use the link:https://github.com/operator-framework/operator-metering/tree/master/Documentation[upstream documentation] to install, configure, and use metering.
 
 To get started installing metering, first install the Metering Operator from OperatorHub. Next, configure your instance of metering by creating a `CustomResource`, referred to here as your MeteringConfig. The installation process creates a default MeteringConfig that you can modify using the examples in the documentation. After creating your MeteringConfig, install the metering stack. Last, verify your installation.
 
+Metering leverages a SQL database called Presto.
+Presto does not store the data itself. Instead, its storage is decoupled from it, often in the form of object storage.
+This means at a minimum you need to configure persistent storage to hold reporting data, and the Hive metastore to hold metadata about database tables managed by Presto and Hive.
```

Comment:
I might rework this paragraph to make it less focused on Presto and more focused on the additional installation requirements.

Reply:
Good call. Actually, I'm going to remove this paragraph because this is a quick overview of the installation steps. The important point is "at a minimum you need to configure persistent storage to hold reporting data, and the Hive metastore to hold metadata about database tables managed by Presto and Hive.", and that is covered in detail when you install the metering stack.